src/generator/patchman.integration.js

```diff
@@ -64,6 +64,22 @@ describe('patchman', function () {
         expect(res.text).toMatchSnapshot();
     });
 
+    test('generates a playbook with patchman advisory remediation with a 6-digit id', async () => {
```

Comment:
Not sure if this test name is too literal 🤔

Reply:
The test name is great - some of the tests have names that are so terse it's hard to tell what they actually do without looking at the test code.